api/client/samples/analogous_years/lib/get_transform_data.py

```diff
@@ -26,6 +26,8 @@ def get_data(client, metric_id, item_id, region_id, source_id, frequency_id, sta
                    'source_id': source_id,
                    'frequency_id': frequency_id,
                    'start_date': start_date}
+    client._data_series_list = []
+    client._data_frame = pd.DataFrame()
```

Comment:
client is passed as an argument; so should I do `client_1 = client` and then work with `client_1`?

Reply:
Discussed offline. Will not client as an argument here.

@nemozen , question should that be part of a different PR. Since, the main problem that I am trying to solve here is not related to this one?

Reply:
ok